src/duplicacy_gcdstorage.go

```diff
@@ -62,8 +71,9 @@ func (storage *GCDStorage) shouldRetry(threadIndex int, err error) (bool, error)
 			retry = true
 		} else if e.Code == 403 {
 			// User Rate Limit Exceeded
-			message = "User Rate Limit Exceeded"
+			message = e.Message // "User Rate Limit Exceeded"
```

Comment:
Looks like you accidentally left the previous message commented out; please remove.